lib/settings/settings.js

```diff
@@ -138,6 +138,7 @@ class Settings {
    */
   initSettingsObject() {
     this.settings = Settings.getDefaults();
+    this.settings.testEnv = this.testEnv;
```

Comment:
testEnv is used in Results instance. Can't we just send it when the Results instance is created?